docs/DevGuide/Notation.md

```diff
@@ -0,0 +1,127 @@
+  \f{align*}{
+      \mathbf{J}=\frac{\partial \mathbf{x}}{\mathbf{\xi}}
+  \f}
+  where \f$\mathbf{x}\f$ are the coordinates mapped to by coordinate map.
+  The determinant of the Jacobian matrix is denoted by \f$J\f$.
+- We use \f$s\f$ and \f$t\f$ to be grid point indices. That is, \f$U_s\f$ is
+  \f$U\f$ at the grid point \f$s\f$. For tensor product bases we index the
+  individual irreducible topologies by subscripting \f$s\f$. For example,
+  \f$s_1\f$ would be the \f$\xi\f$-dimension in the reference element.
+- We use a nodal basis of Lagrange interpolating polynomials
+  \f{align*}{
+      \ell_{s_i}(\xi)=\prod^{N}_{\substack{t_i=0\\ t_i\ne s_i}}
+      \frac{\xi - \xi_{t_i}}{\xi_{s_i}-\xi_{t_i}},
+  \f}
+  where \f$N\f$ is the order/degree of the basis in each logical direction.
+- A DG scheme with an \f$N^{\mathrm{th}}\f$ order basis is referred to as a
+  \f$P_N\f$ scheme and converges at order \f$\mathcal{O}\left(\Delta
+  x^{N+1}\right)\f$ where \f$\Delta x\f$ is the 1d size of the element.
+- The unit normal vector to the element is denoted by \f$n_i\f$.
+- The numerical flux boundary correction term which includes the normal vectors
+  is denoted by \f$G(F^{i,+},n_i^+,u^+,F^{i,-},n_i^-,u^-)\f$ where the \f$+\f$
+  and \f$-\f$ denote the values on the outer and inner side of the mortar,
+  respectively. To give a concrete example, for the Rusanov flux we have
+  \f{align*}{
+      G^{\mathrm{Rusanov}}=
+      \frac{1}{2}\left(F^{i,+}n_i^+-F^{i,-}n_i^-\right)
+      -\frac{C}{2}\left(u^+-u^-\right)
+  \f}
+
+# General Relativity and (Magneto)hydrodynamics{#gr_hydro_notation}
+
+We use the following conventions and notation:
+- In general we work in units where \f$G=c=1\f$.
+- Latin letters at the beginning of the alphabet, \f$a,b,c,\ldots\f$ are
+  spacetime indices ranging from 0 to 3.
+- Latin letters \f$i,j,k,\ldots\f$ are spatial indices ranging from 1 to 3.
+
+The notation for general relativity quantities used is:
+- The Einstein summation convention is used where repeated indices are summed
+  over.
+- The spacetime metric is denoted by \f$g_{ab}\f$ with signature
+  \f$(-,+,+,+)\f$, and its determinant by \f$g\f$.
+- The lapse and shift are denoted by \f$\alpha\f$ and \f$\beta^i\f$,
+  respectively.
+- The spatial metric and its determinant are denoted by \f$\gamma_{ij}\f$ and
+  \f$\gamma\f$, respectively.
+- We write the line element as:
+  \f{align*}{
+      ds^2=g_{ab}dx^a dx^b=-\alpha^2 dt^2 +
+      \gamma_{ij}
+      \left(dx^i + \beta^i dt\right)\left(dx^j + \beta^j dt\right).
+  \f}
+- The unit timelike normal to the hypersurface is
+  \f$n^a=(1/\alpha,-\beta^i/\alpha)\f$ and \f$n_a=(-\alpha,0,0,0)\f$.
+- The extrinsic curvature is denoted by \f$K_{ij}\f$ and its trace by
+  \f$K\f$. We use the sign convention
+  \f{align*}{
+      K_{ab}=-\frac{1}{2}\mathcal{L}_n g_{ab}
+  \f}
+- For the generalized harmonic evolution system \f$\Pi_{ab}=-n^c\partial_c
+  g_{ab}\f$ and \f$\Phi_{iab}=\partial_i g_{ab}\f$.
+
```

Comment:
Could you add the energy-momentum tensor and, in particular, the energy density `E=t_a t_b T^{ab}`?

Reply:
Sure, but it's `n_a` and `n_b`.

Reply:
In https://github.com/sxs-collaboration/spectre/pull/1405#discussion_r264855270 I was told the opposite though. The clash with the face normals is real :smile:

Reply:
Yes, the thing is that SpECTRE is our chance to move to the standard NR notation that is in all the text books. Since the spacetime normal and the element normal show up in completely different contexts I don't think this will be an issue and it is consistent with all the DG literature (maybe except the Jena group who at least in one paper use `s_i` for the element normal). I think it's fine to have this collision. There aren't enough letters to be collision free, unfortunately :P

Reply:
I'm not thrilled with t^a for the spatial normal to an element interface. First, it has so many other connotations. Second, very soon we will want a notation for a tangential vector to the interface. t^a would be a good choice for that. I'm leaning toward s^a for the normal.

Reply:
I don't think anyone is proposing to use t^a as a spatial normal to an element interface.  I thought we had agreed that n_i is the spatial normal to an element interface.